platform/ios/core-files.txt

```diff
@@ -0,0 +1,166 @@
+platform/darwin/src/MGLDistanceFormatter.m
+
+# SDK/Foundation/Geometry
+platform/darwin/src/MGLFeature.mm
+platform/darwin/src/MGLGeometry.mm
+platform/darwin/src/MGLMultiPoint.mm
+platform/darwin/src/MGLPointAnnotation.mm
+platform/darwin/src/MGLPointCollection.mm
+platform/darwin/src/MGLPolygon.mm
+platform/darwin/src/MGLPolyline.mm
+platform/darwin/src/MGLShape.mm
+platform/darwin/src/MGLShapeCollection.mm
+
+# SDK/Foundation/Offline Maps
+platform/darwin/src/MGLOfflinePack.mm
+platform/darwin/src/MGLOfflineStorage.mm
+platform/darwin/src/MGLShapeOfflineRegion.mm
+platform/darwin/src/MGLTilePyramidOfflineRegion.mm
+
+# SDK/Foundation/Styling
+platform/darwin/src/MGLLight.mm
+platform/darwin/src/MGLStyleValue.mm
+
+# SDK/Foundation/Styling/Categories
+platform/darwin/src/NSValue+MGLStyleAttributeAdditions.mm
+
+# SDK/Foundation/Styling/Layers
+platform/darwin/src/MGLBackgroundStyleLayer.mm
+platform/darwin/src/MGLCircleStyleLayer.mm
+platform/darwin/src/MGLFillExtrusionStyleLayer.mm
+platform/darwin/src/MGLFillStyleLayer.mm
+platform/darwin/src/MGLForegroundStyleLayer.mm
+platform/darwin/src/MGLHeatmapStyleLayer.mm
+platform/darwin/src/MGLHillshadeStyleLayer.mm
+platform/darwin/src/MGLLineStyleLayer.mm
+platform/darwin/src/MGLOpenGLStyleLayer.mm
+platform/darwin/src/MGLRasterStyleLayer.mm
+platform/darwin/src/MGLStyleLayer.mm
+platform/darwin/src/MGLSymbolStyleLayer.mm
+platform/darwin/src/MGLVectorStyleLayer.m
+
+# SDK/Foundation/Styling/Sources
+platform/darwin/src/MGLComputedShapeSource.mm
+platform/darwin/src/MGLImageSource.mm
+platform/darwin/src/MGLRasterDEMSource.mm
+platform/darwin/src/MGLRasterTileSource.mm
+platform/darwin/src/MGLShapeSource.mm
+platform/darwin/src/MGLSource.mm
+platform/darwin/src/MGLTileSource.mm
+platform/darwin/src/MGLVectorTileSource.mm
+
+# SDK/Kit
+platform/ios/src/MGLMapAccessibilityElement.mm
+platform/ios/src/MGLMapView.mm
+
+# SDK/Kit/Annotations
+platform/ios/src/MGLAnnotationContainerView.m
+platform/ios/src/MGLAnnotationImage.m
+platform/ios/src/MGLAnnotationView.mm
+platform/ios/src/MGLCompactCalloutView.m
+platform/ios/src/MGLFaux3DUserLocationAnnotationView.m
+platform/ios/src/MGLUserLocation.m
+platform/ios/src/MGLUserLocationAnnotationView.m
+platform/ios/src/MGLUserLocationHeadingArrowLayer.m
+platform/ios/src/MGLUserLocationHeadingBeamLayer.m
+
+# SDK/Kit/Categories
+platform/darwin/src/NSCoder+MGLAdditions.mm
+platform/ios/src/NSOrthography+MGLAdditions.m
+platform/ios/src/UIColor+MGLAdditions.mm
+platform/ios/src/UIDevice+MGLAdditions.m
+platform/ios/src/UIImage+MGLAdditions.mm
+platform/ios/src/UIViewController+MGLAdditions.m
+
+# SDK/Kit/SMCalloutView
+platform/ios/vendor/SMCalloutView/SMCalloutView.m
+
+# SDK/Kit/Telemetry
+platform/ios/src/MGLTelemetryConfig.m
+
+# SDK/Kit/Telemetry/Development
+platform/ios/src/MGLSDKUpdateChecker.mm
+
+# SDK/Kit/Telemetry/Runtime
+platform/ios/src/MGLMapboxEvents.m
+platform/ios/vendor/mapbox-events-ios/MapboxMobileEvents/CLLocation+MMEMobileEvents.m
```

Comment:
If the events library’s files are being inlined here, that means the script needs to be rerun whenever the submodule pin changes.

Reply:
Correct